### DISCUSSION TITLE
2021.02.03 ActiveSync on Android 12 just deletes contacts

### DISCUSSION THREAD

**Discussion by DerBergRuft** [tine20/tine20#7362] [Q&A]
I am using Tine 2.0 CE 2021.02.03 with ActiveSync for contacts enabled on a Debian 9 root server. On my Windows 10 PC with Thunderbird + TbSync I can sync all contacts in both directions. On my phone Galaxy S20 FE with Android 12 I set up the Exchange Account. When the phone is syncing contacts there is no error message. Contatcs on my phone bound to the Exchange Account disappear after sync, contacts from my Tine addressbook never show up on the phone. The apache log contains commands 'FolderSync', 'Settings', 'Sync' without any errors or warnings.
Can someone please provide directions to debug the issue?
Thank you

**Comment by lab-at-nohl** [comment]
Set the logger to level DEBUG to get more information. Also try to delete the device in Admin before set the account up on Android again. 

I experienced that Android stopps working if a single contact is faulty for whatever reason. But never that contacts are really deleted.

**Reply by DerBergRuft** [reply]
Where do you want me to set a log to debug level? Apache? Tine20? Somewhere else? I set up mod_security for my apache to log message bodies. I didn't give me any useful hint. Now I use OpenSync on my phone to sync contacts with tine20 via CardDAV. That works, ActiveSync seems to be dead.

**Reply by lab-at-nohl** [reply]
Tine, either setup.php or directly in config.inc.php (see wiki). Have you tried to remove the device as admin?

**Reply by DerBergRuft** [reply]
I found this array in config.inc.php
  'logger' =>
  array (
    'active' => true,
    'filename' => '/var/log/tine20/tine20.log',
    'priority' => '3',
  ),
but no description of possible priorities.

Deleting the device from the list of activesync devices did not solve the issue.

**Reply by pschuele** [reply]
hi @DerBergRuft ,
sorry for the late reply ... are you still here?

the log level should be set to 7 (DEBUG). the possible levels can be seen when going to your.tine.com/setup.php (check this page for description of the levels: https://en.wikipedia.org/wiki/Syslog#Severity_level).
